src/EditorFeatures/CSharpTest/Completion/CompletionProviders/PropertySubPatternCompletionProviderTests.cs

```diff
@@ -503,7 +503,7 @@ class D
     public int P2 { get; set; }
 }
 ";
-            await VerifyItemExistsAsync(markup, "P1");
+            await VerifyNoItemsExistAsync(markup);
```

Comment:
Did this sneak in from another unrelated PR?

Reply:
No, just fixing a test broken in the merged branch. Probably broken in master too. See https://github.com/dotnet/roslyn/pull/45990#issuecomment-658483390